include/wlr/types/wlr_idle_inhibit_v1.h

```diff
@@ -0,0 +1,33 @@
+#ifndef WLR_TYPES_WLR_IDLE_INHIBIT_V1_H
+#define WLR_TYPES_WLR_IDLE_INHIBIT_V1_H
+
+#include <wayland-server.h>
+
+struct wlr_idle_inhibit_manager_v1 {
+	struct wl_list wl_resources; // wl_resource_get_link
+	struct wl_list inhibitors; // wlr_idle_inhibit_inhibitor_v1::link
+	struct wl_global *global;
+	
+	struct wl_listener display_destroy;
+
+	struct {
+		struct wl_signal new_inhibitor;
+	} events;
+};
+
+struct wlr_idle_inhibit_inhibitor_v1 {
```

Comment:
The interface name is:

```xml
<interface name="zwp_idle_inhibitor_v1" version="1">
```

So this should be named `wlr_idle_inhibitor_v1`.

Reply:
The guidelines say to use `wlr_submodule_<name>` for typenames.
This is the `idle_inhibit_v1` submodle. So we we not use the same naming conventions for protocol implementations? If that's the case, the proper naming scheme should probably be added to the guidelines.